jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>Not a helpful comment, but I think this example is a bit terrible because you're comparing two timepoints and it doesn't make much biological sense for you to say 'Microglia had the biggest difference in differences between timepoints." If I were to point out something interesting about this example, I would rather say that it's more interesting to find a cell type that is NOT so different between the two timepoints, which indicates the some permanent damage was done which was not recovered from, or some conclusion like this!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>